developersample/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
oops

---
In reply to: [114930426](https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/165#discussion_r114930426) [](ancestors = 114930426)